hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java

```diff
@@ -3942,6 +3942,7 @@ public static class StatisticsData {
       private volatile long bytesReadDistanceOfThreeOrFour;
       private volatile long bytesReadDistanceOfFiveOrLarger;
       private volatile long bytesReadErasureCoded;
+      private volatile long remoteBytesReadTime;
```

Comment:
Can we rename to include unit of time in the variable? `remoteBytesReadTimeSecs` or `remoteBytesReadTimeMS`

Reply:
Good suggestion. modified